cp-ksqldb-cli/Dockerfile.ubi8

```diff
@@ -53,7 +53,12 @@ gpgcheck=1 \n\
 gpgkey=${CONFLUENT_PACKAGES_REPO}/archive.key \n\
 enabled=1 " > /etc/yum.repos.d/confluent.repo \
     && echo "===> Installing Confluent Hub client ..." \
-    && yum install -y confluent-hub-client-${CONFLUENT_VERSION} \
+    && yum install -y \
```

Comment:
I am not sure we need this for the CLI... 🤔

Reply:
yea, let's not install this for the CLI (we also don't install `confluent-security`)